Assets/Scripts/Dungeon/DungeonGenerator.cs

```diff
@@ -66,25 +74,25 @@ IEnumerator GenerateDungeon()
 
             int randomOffset = rand.Next(maxOffset * 2) - maxOffset;
```

Comment:
Comment what the upcomming sequence does please. I do understand what happens, but I had to look pretty damn carefully to understand